src/index.js

```diff
@@ -0,0 +1,159 @@
+    axios.get('http://localhost:5000/weather', {
+        params: {lat: state.latitude, lon: state.longitude}
+    }) 
+    .then(response => {
+    // check out how a response looks: https://openweathermap.org/current#geo or the replit I shared
+    const weather = response.data.main.temp
+    // couldn't get adding a unit (fahrenheit) param to work for temperature, there is an option in the docs
+    state.temp = Math.round(convertKtoF(weather));
+    changeTempColorAndLandscape();
+    })
+    .catch(error =>{
+        console.log("Weather error: ", error.response.data)
+    });
+
+}
+
+//functions to increase and decrease the temp
+const increaseTemp = () => {
+    state.temp += 1;
+    changeTempColorAndLandscape()
+  }
+const decreaseTemp = () => {
+    state.temp -= 1;
+    changeTempColorAndLandscape()
+  }
+
+//function to change sky
+const changeSkyDisplay = () => {
+    //getting selected sky image
+  const input = document.getElementById('sky_select').value;
+// selecting the sky_image element object
+  const currentSky = document.getElementById('sky_image');
+  let skyDisplay =	'🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+
+  if (input === 'Sunny'){
+    skyDisplay =	'☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+  }else if (input === 'Cloudy'){
+    skyDisplay =	'☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+  }else if (input === 'Rainy'){
+    skyDisplay =	'🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+  }else if (input === 'Snowy'){
+    skyDisplay =	'🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+  };
+//   skyImage.textContent = skyDisplay;
+  currentSky.textContent = skyDisplay;
+};
+
+//function to change the landscape and temperature value color
+const changeTempColorAndLandscape = () => {
+    let temperature_value = state.temp;
+    let temperature_color = 'yellow';
+    let landscape =	"🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+
+    if (temperature_value > 80) {
+        landscape = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+        temperature_color = 'red'
+    } else if (temperature_value >= 70) {
+        landscape = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+        temperature_color = 'orange'
+    } else if (temperature_value >= 60) {
+        landscape = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+        temperature_color = 'yellow'
+    } else if (temperature_value >= 50) {
+        landscape = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+        temperature_color = 'green'
+    } else {
+        landscape = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+        temperature_color = 'teal'
+    }
+
+    // First, get element we want to alter
+    // Next, set the textcontent (the string that represents the image) of that element
+    const currentLandscape = document.getElementById('landscape_image');
+    currentLandscape.textContent = landscape;
+    // First, get element we want to alter
+    // Change the classname according to the color of the range
+    // Next, set the textcontent (the value literal) of that element, to be the value of the integer in current state casted to a string
+    const tempValue = document.getElementById('temperature-value');
+    tempValue.className = temperature_color;
+    tempValue.textContent = String(state.temp);
```

Comment:
another way to create a string, without using the String constructor is to just use a template string:

```
`${state.temp}` 
```